### PR TITLE
Create events

### PR DESCRIPTION
**Pull request by dipscoder**
@Abhishek-kumar09 

User can create events with image upload

### CODE REVIEW

File: src/components/CreateNewEvent.js [hunk 1]
```diff
@@ -40,7 +44,10 @@ const useStyles = makeStyles(theme => ({
     width: '200px',
     height: '38px',
     borderRadius: '20px',
-    marginRight: '15px'
+    marginRight: '15px',
+    '&:hover': {
+      backgroundColor: '#101c4c'
+    }
```
Comment (Abhishek-kumar09) [inline src/components/CreateNewEvent.js]: remove hover

File: src/components/CreateNewEvent.js [hunk 2]
```diff
@@ -56,7 +63,10 @@ const useStyles = makeStyles(theme => ({
     width: '200px',
     height: '38px',
     borderRadius: '20px',
-    marginRight: '60px'
+    marginRight: '60px',
+    '&:hover': {
+      backgroundColor: '#101c4c'
+    }
```
Comment (Abhishek-kumar09) [inline src/components/CreateNewEvent.js]: remove hover

File: src/components/CreateNewEvent.js [hunk 3]
```diff
@@ -145,6 +155,9 @@ const useStyles = makeStyles(theme => ({
     fontWeight: 'bold',
     fontSize: '13px',
     background: '#291757',
+    '&:hover': {
+      backgroundColor: '#101c4c'
+    },
```
Comment (Abhishek-kumar09) [inline src/components/CreateNewEvent.js]: here too

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]
hover effect has been fixed in latest code, remove the `:hover` from your code

File: src/components/CreateNewEvent.js [hunk 4]
```diff
@@ -193,18 +206,74 @@ const useStyles = makeStyles(theme => ({
...
+    let storageRef = firebase.storage().ref();
+    let fileRef = storageRef.child(`${userId}/${file.name}`);
+    await fileRef.put(file);
+    setImageURL(await fileRef.getDownloadURL());
+  };
+
+  const handleSubmit = () => {
+    let userId = user.uid;
+    formData.speakers = speaker;
+    formData.userId = userId;
```
Comment (Abhishek-kumar09) [inline src/components/CreateNewEvent.js]: change `formData.userId` to `formData.createdBy`

**Comment by dipscoder** [comment]
Ok

**Pull request merged**